CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # CHANGELOG
```

Comment:
I'm missing changes in `src/webview-bridge-cjs.js.flow`

That file contains flow type definitions (flow is the typesystem we use in webapp)

Reply:
You are right. Thanks for the clarification.